runelite-client/src/main/java/net/runelite/client/plugins/coxhelper/CoxPlugin.java

```diff
@@ -217,9 +217,16 @@ private void onChatMessage(ChatMessage event)
 			{
 				for (Player player : client.getPlayers())
 				{
-					if (player.getName().equals(tpMatcher.group(1)))
+					final String rawPlayerName = player.getName();
+
+					if (rawPlayerName != null)
 					{
-						victims.add(new Victim(player, Victim.Type.TELEPORT));
+						final String fixedPlayerName = Text.sanitize(rawPlayerName);
```

Comment:
Text.sanitize just removes images from names, for instance, the hardcore ironman symbol when someone talks. A better option would be text.standardize, or text.toJagexName